public/entity2/concreteentitylist.h

```diff
@@ -0,0 +1,28 @@
+#ifndef CONCRETEENTITYLIST_H
+#define CONCRETEENTITYLIST_H
+
+#include "entityidentity.h"
+
+class CConcreteEntityList
+{
+	struct CList
+	{
+		CEntityIdentity* m_pHead;
+		CEntityIdentity* m_pTail;
+		uint64_t unknown;
```

Comment:
Is this ``unknown`` correct? I have suspicions that this is incorrectly assumed based on the base structure being incorrectly padded, see comments below.

Reply:
Yeah, it's not just padding because it's getting increased/decrease by 1 every time there's some operation related to the head/tail variable. I assume it's some sort of size tracker for the thing?